layouts/index.html

```diff
@@ -40,7 +40,7 @@
 								<a href="{{ $postsPaging.Prev.URL | absURL }}" class="previous">Prev</a>
 								{{ end }}
 								{{ range $postsPaging.Pagers }}
-								{{ if lt .PageNumber 10 }}
+								{{ if lt .PageNumber 15 }}
```

Comment:
See my comments on the issue regarding this fix:

https://github.com/curttimson/hugo-theme-massively/issues/23#issuecomment-382411305

This would only fix the issue up to 15 pages. A better solution, in my opinion, would be to make the pages displayed relative to the current page which you are on.